cpp/src/binaryop/jit/code/code.h

```diff
@@ -16,8 +19,18 @@
 
 #pragma once
 
-#define _LIBCUDACXX_USE_CXX20_CHRONO
-#define _LIBCUDACXX_USE_CXX17_TYPE_TRAITS
+namespace cudf {
+namespace experimental {
+namespace binops {
+namespace jit {
+namespace code {
+
+    extern const char* kernel;
+    extern const char* traits;
+    extern const char* operation;
 
-#include <simt/chrono>
```

Comment:
This deletion of `chrono.hpp` also looks like a merge issue but it could very well be a conscious decision. Seems like you had added a `chrono.hpp` in some previous PR but then you would've also had to stringify that header for this PR, so you removed it?
If my understanding is correct, then ignore this review comment.

Reply:
Yeah I removed it so we wouldn't have to worry about stringifying another header.